tests/session/transaction-integration-001.phpt

```diff
@@ -20,12 +20,20 @@ $manager = new MongoDB\Driver\Manager(URI);
 $cmd = new \MongoDB\Driver\Command([
     'create' => $EMPLOYEES_COL
 ]);
-$manager->executeCommand(DATABASE_NAME, $cmd);
+$manager->executeCommand(
+    DATABASE_NAME,
+    $cmd,
```

Comment:
If we're splitting `executeCommand()` up into multiple lines, I think we can easily inline the command itself, too.

Suggest replacing `$cmd` with `new \MongoDB\Driver\Command([ 'create' => $EMPLOYEES_COL ])`.